controllers/managedfusionoffering_controller.go

```diff
@@ -181,18 +189,41 @@ func (r *ManagedFusionOfferingReconciler) reconcileSubscription() error {
 	return nil
 }
 
+func (r *ManagedFusionOfferingReconciler) addManagedFusionOfferingAnnotation(obj metav1.Object, key string) {
+	annotations := obj.GetAnnotations()
+	if annotations == nil {
+		annotations = map[string]string{}
+		obj.SetAnnotations(annotations)
+	}
+	value := fmt.Sprintf("%s/%s", r.managedFusionOffering.Namespace, r.managedFusionOffering.Name)
+	annotations[key] = value
+}
```

Comment:
We have a util for adding annotations so why are we implementing this here?

Reply:
Updated